cabby/data/metagraph/utils.py

```diff
@@ -169,75 +144,81 @@ def convert_multidi_to_weighted_undir_graph(
   Returns:
     out_graph: graph with weighted undirected edges.
   """
+  edges = collections.defaultdict(dict)
+  for node, neighbor_dict in in_graph.adjacency():
+    for neighbor, edges_dict in neighbor_dict.items():
+      id_pair = tuple(sorted([node, neighbor]))
+      if 'weight' not in edges[id_pair]:
+        edges[id_pair]['weight'] = 0.0
+      for _, edge_data in edges_dict.items():
+        if 'true_length' in edge_data:
+          weight = edge_data['true_length']
+        else:
+          weight = edge_data['length']
+        edges[id_pair]['weight'] += weight
```

Comment:
What is the scale on these weights? I think it is tens of meters? (Would be good to document briefly.)

Reply:
Not sure, but as we discussed off-thread, we want this to apply to all scales. I left a TODO to add more edge weight details in e2087a6